src/routes/safe/components/Apps/ManageApps.js

```diff
@@ -9,7 +9,7 @@ import { getAppInfoFromUrl } from './utils'
 import Field from '~/components/forms/Field'
 import DebounceValidationField from '~/components/forms/Field/DebounceValidationField'
 import GnoForm from '~/components/forms/GnoForm'
-import { composeValidators, required } from '~/components/forms/validator'
+import { composeValidatorsApps, required } from '~/components/forms/validator'
```

Comment:
if this validator is used only here, I would place it there

Reply:
done